Assets/Scripts/Tools/AreaSelection.cs

```diff
@@ -1,11 +1,12 @@
-using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.InteropServices;
+// ReSharper disable once RedundantUsingDirective
 using System.Text;
 using Netherlands3D.Collada;
 using Netherlands3D.MeshClipping;
+using UnityEditor;
```

Comment:
can you add `#if UNITY_EDITOR` around this using?

Reply:
Absolutely!